### PR TITLE
fixes url for hrefs and links in text in both english and spanish for 03-back-end-development-certification

### LINKED ISSUE #9679: Reference to lesson "get-set-for-our-back-end-development-projects" does not exist
Challenge [URL Shortener Microservice](https://www.freecodecamp.com/challenges/url-shortener-microservice) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

There is a link to [get-set-for-our-back-end-development-projects](https://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects) on this challenge and also on several others, but it does not exist

### PR DESCRIPTION
**Pull request by Vince33**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #9679
#### Description

I swapped out the urls for the updated url in the map modal for all related  exercises

### CODE REVIEW

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 1]
```diff
@@ -160,7 +160,7 @@
       "title": "URL Shortener Microservice",
       "description": [
         "<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='https://little-url.herokuapp.com/' target='_blank'>https://little-url.herokuapp.com/</a> and deploy it to Heroku.",
-        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-back-end-development-projects'>https://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-api-development-projects'>https://www.freecodecamp.com/challenges/get-set-for-our-api-development-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: You don't need to add `www.` here.

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 2]
```diff
@@ -194,7 +194,7 @@
...
       "description": [
         "<strong>Objective:</strong> Build a full stack JavaScript app that allows you to search for images like this: <a href='https://cryptic-ridge-9197.herokuapp.com/api/imagesearch/lolcats%20funny?offset=10' target='_blank'>https://cryptic-ridge-9197.herokuapp.com/api/imagesearch/lolcats%20funny?offset=10</a> and browse recent search queries like this: <a href='https://cryptic-ridge-9197.herokuapp.com/api/latest/imagesearch/' target='_blank'>https://cryptic-ridge-9197.herokuapp.com/api/latest/imagesearch/</a>. Then deploy it to Heroku.",
-        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-back-end-development-projects'>https://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-api-development-projects'>https://www.freecodecamp.com/challenges/get-set-for-our-api-development-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: You don't need to add `www.` here.

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 3]
```diff
@@ -227,7 +227,7 @@
       "title": "File Metadata Microservice",
       "description": [
         "<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='https://aryanj-file-size.herokuapp.com/' target='_blank'>https://aryanj-file-size.herokuapp.com/</a> and deploy it to Heroku.",
-        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-back-end-development-projects'>https://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-api-development-projects'>https://www.freecodecamp.com/challenges/get-set-for-our-api-development-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: You don't need to add `www.` here.

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 4]
```diff
@@ -191,7 +191,7 @@
       "title": "Build a Voting App",
       "description": [
         "<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='https://fcc-voting-arthow4n.herokuapp.com/' target='_blank'>https://fcc-voting-arthow4n.herokuapp.com/</a> and deploy it to Heroku.",
-        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 5]
```diff
@@ -212,7 +212,7 @@
       "challengeType": 4,
       "descriptionEs": [
         "<strong>Objetivo:</strong> Construye una aplicación de pila completa (full stack) en JavaScript que funcione de forma similar al siguiente proyecto: <a href='https://fcc-voting-arthow4n.herokuapp.com/' target='_blank'>https://fcc-voting-arthow4n.herokuapp.com/</a> y despliégala en Heroku.",
-        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 6]
```diff
@@ -233,7 +233,7 @@
       "title": "Build a Nightlife Coordination App",
       "description": [
         "<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='http://whatsgoinontonight.herokuapp.com/' target='_blank'>http://whatsgoinontonight.herokuapp.com/</a> and deploy it to Heroku.",
-        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 7]
```diff
@@ -251,7 +251,7 @@
       "challengeType": 4,
       "descriptionEs": [
         "<strong>Objetivo:</strong> Construye una aplicación de pila completa (full stack) en JavaScript que funcione de forma similar al siguiente proyecto: <a href='http://whatsgoinontonight.herokuapp.com/' target='_blank'>http://whatsgoinontonight.herokuapp.com/</a> y despliégala en Heroku.",
-        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 8]
```diff
@@ -269,7 +269,7 @@
       "title": "Chart the Stock Market",
       "description": [
         "<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='http://watchstocks.herokuapp.com/' target='_blank'>http://watchstocks.herokuapp.com/</a> and deploy it to Heroku.",
-        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 9]
```diff
@@ -286,7 +286,7 @@
       "challengeType": 4,
       "descriptionEs": [
         "<strong>Objetivo:</strong> Construye una aplicación de pila completa (full stack) en JavaScript que funcione de forma similar al siguiente proyecto: <a href='http://watchstocks.herokuapp.com/' target='_blank'>http://watchstocks.herokuapp.com/</a> y despliégalo en Heroku.",
-        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 10]
```diff
@@ -303,7 +303,7 @@
       "title": "Manage a Book Trading Club",
       "description": [
         "<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='http://bookjump.herokuapp.com/' target='_blank'>http://bookjump.herokuapp.com/</a> and deploy it to Heroku.",
-        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 11]
```diff
@@ -320,7 +320,7 @@
       "challengeType": 4,
       "descriptionEs": [
         "<strong>Objetivo:</strong> Construye una aplicación de pila completa (full stack) en JavaScript que funcione de forma similar al siguiente proyecto: <a href='http://bookjump.herokuapp.com/' target='_blank'>http://bookjump.herokuapp.com/</a> y despliégalo en Heroku.",
-        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 12]
```diff
@@ -337,7 +337,7 @@
       "title": "Build a Pinterest Clone",
       "description": [
         "<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='http://pintech.herokuapp.com/' target='_blank'>http://pintech.herokuapp.com/</a> and deploy it to Heroku.",
-        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Note that for each project, you should create a new GitHub repository and a new Heroku project. If you can't remember how to do this, revisit <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 13]
```diff
@@ -357,7 +357,7 @@
       "challengeType": 4,
       "descriptionEs": [
         "<strong>Objetivo:</strong> Construye una aplicación de pila completa (full stack) en JavaScript que funcione de forma similar al siguiente proyecto: <a href='http://pintech.herokuapp.com/' target='_blank'>http://pintech.herokuapp.com/</a> y despliégalo en Heroku.",
-        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com/challenges/get-set-for-our-back-end-development-projects</a>.",
+        "Ten en cuenta que para cada proyecto, debes crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerdas cómo hacerlo, visita de nuevo <a href='/challenges/get-set-for-our-dynamic-web-application-projects'>http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects</a>.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- http://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
+ https://freecodecamp.com/challenges/get-set-for-our-dynamic-web-application-projects
```

**Comment by Vince33** [comment]
ok so about to  remove the www s  I was not clear on the other comments and notes looking for further clarification before i append to the commit

**Comment by Vince33** [comment]
I think i see the http needs to be changed to https in a number of cases

**Comment by atjonathan** [comment]
@Vince33, the other comments were just about replacing `http` with `https` :

**Comment by Vince33** [comment]
I appended and force updated to the branch changes should be there

**Comment by atjonathan** [comment]
#### :sparkles: :sparkles: **LGTM :+1:** :sparkles: :sparkles: 

:tada: Thanks for contributing @Vince33 and congratulations on your first PR submission to FreeCodeCamp! :tada:

**Pull request merged**